core/pom.xml

```diff
@@ -215,6 +215,11 @@
       <version>3.2.10</version>
     </dependency>
     <dependency>
+      <groupId>com.fasterxml.jackson.module</groupId>
+      <artifactId>jackson-module-scala_2.10</artifactId>
+      <version>2.3.1</version>
```

Comment:
this is rxin: ah i c we are already adding it. the question stands for @pwendell / @andrewor14 - should we shade this? Is it robust enough to not shade?